src/main/java/org/broadinstitute/consent/http/health/SendGridStatus.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.google.gson.Gson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SendGridStatus {
+    private Object page;
+    private Object status;
+
+    public Object getPage() {
+        return page;
+    }
+
+    public void setPage(Object page) {
+        this.page = page;
+    }
+
+    public Object getStatus() {
+        return status;
+    }
+
+    public void setStatus(String indicator, String description) {
+        Map<String, String> statusMap = new HashMap<>();
+        statusMap.put("indicator", indicator);
+        statusMap.put("description", description);
+        status = new Gson().toJson(statusMap);
```

Comment:
See comment about what `status` really should be. In this example, status is just a `String`, which adds unnecessary complexity in your private methods `getStatusMap()` and `isOk()`